python/src/scipp/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from ._scipp.core import *
 from ._scipp import __version__
+from ._scipp import neutron
```

Comment:
Shouldn't this be `from . import neutron`? Otherwise you will not see the Python bits.